### DISCUSSION TITLE
No way to change tyre compound

### DISCUSSION THREAD

**Discussion by Liam-Haycock** [kutu/pyirsdk#65] [General]
is there anyway to find out what tyre compounds are available for the cars in pitstops and change to a different compound (e.g. wets softs, or dries)

**Comment by kutu** [comment]
there is in `DriverInfo`
<img width="247" height="172" alt="image" src="https://github.com/user-attachments/assets/45099a34-7d61-4f6d-88b8-ce3b73c52060" />

and then you send pit command with tyre index
<img width="358" height="293" alt="image" src="https://github.com/user-attachments/assets/9de8cda7-5baf-4c55-a8af-30b22934039c" />
